csharp_dotnetcore/17.Proactive-Messages/README.md

```diff
@@ -0,0 +1,41 @@
+This sample demonstrates how to send proactive messages to users by
+capturing a conversation reference, then using it later to initialize
+outbound messages using ASP.Net Core 2.
+
+# Concepts introduced in this sample
+**Proactive messaging**. Often bots send _reactive messages_, a message based on the user's last input. However, we might need to send a proactive message, one not based on the user's input.
+
```

Comment:
strongly consider using the existing docs here (cut/paste). 
[https://docs.microsoft.com/en-us/azure/bot-service/nodejs/bot-builder-nodejs-proactive-messages?view=azure-bot-service-3.0](https://docs.microsoft.com/en-us/azure/bot-service/nodejs/bot-builder-nodejs-proactive-messages?view=azure-bot-service-3.0)

Reading the text here it's still not clear what a proactive message actually is. The existing v3 docs do a cleaner job of explaining this.

Reply:
Heh, I think I lifted this from v3, actually replaced with v4..